### DISCUSSION TITLE
LFP or NMC Battery option in setup

### DISCUSSION THREAD

**Discussion by dyter69** [teslamate-org/teslamate#3752] [Ideas]
Hello,

I suggest an option for select type of battery to avoid having duplicates menu "battery health", "charging stats" and "overview"

**Comment by jheredianet** [comment]
I second this idea, in the meantime I'm suggesting to remove those dashboards https://github.com/teslamate-org/teslamate/pull/3954

**Comment by swiffer** [comment]
yes, see here ->

https://github.com/teslamate-org/teslamate/issues/3524#issuecomment-1850452981

should be a per car setting.
